cypress/TestCases/FireboltCertification/AdditionalTestcases/IntegratedPlayer.feature

```diff
@@ -0,0 +1,64 @@
+Feature: IntegratedPlayer
+
+    @initialization
+    Scenario: Launch FCA for 'IntegratedPlayer'
+        Given the environment has been set up for 'IntegratedPlayer' tests
+        And 3rd party 'certification' app is launched with 'inactive' state
+
+    @sdk @IntegratedPlayer
+    Scenario: Positive Scenario : Create and validate broadcast player instance
+        When 1st party app invokes the 'Firebolt' API to 'create broadcastPlayer instance'
+        Then 'Firebolt' platform responds to '1st party app' with 'broadcastPlayer instance'
+        When 1st party app invokes the 'Firebolt' API to 'get status of broadcastPlayer'
```

Comment:
also we can place TC first before creating a player . 

- So suppose if we call player.load, player.play without creating a broadcast/streaming player, will the platform throw error ?

or what does player.destroy do ? 
will it destroy the player ?  can you confirm that?